remove git commands from build scripts

John Dey, 2019-06-06 jfdey@fredhutch.org fizwit@github.com
diff -ruN revbayes.org/projects/cmake/generate_version_number.sh revbayes/projects/cmake/generate_version_number.sh
--- revbayes.org/projects/cmake/generate_version_number.sh	2019-06-06 14:12:06.853632000 -0700
+++ revbayes/projects/cmake/generate_version_number.sh	2019-06-06 14:15:10.670528000 -0700
@@ -3,6 +3,6 @@
 echo "Generating version numbers in GitVersion.cpp"
 echo "#include \"GitVersion.h\"" > GitVersion.cpp
 #echo "const char *build_git_sha = \"$(git rev-parse HEAD)\";" >> GitVersion.cpp
-echo "const char *build_git_sha = \"$(git describe --abbrev=6 --always)\";" >> GitVersion.cpp
+echo "const char *build_git_sha = \"fedc517\";" >> GitVersion.cpp
 echo "const char *build_date = \"$(date)\";" >> GitVersion.cpp
-echo "const char *build_git_branch = \"$(git name-rev --name-only HEAD)\";" >> GitVersion.cpp
+echo "const char *build_git_branch = \"v1.0.11\";" >> GitVersion.cpp
